<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvatarSettingsForm</class>
 <widget class="QWidget" name="AvatarSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="useThemeColorsLabel">
      <property name="text">
       <string>Use theme colors</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="useThemeColorsCheckBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="textColorLabel">
      <property name="text">
       <string>Text color</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="textColorLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="textColorToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="backgroundColorLabel">
      <property name="text">
       <string>Background color</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="backgroundColorLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="backgroundColorToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="sizeLabel">
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="sizeSpinBox"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="typeLabel">
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="typeComboBox">
      <item>
       <property name="text">
        <string>Letter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Image</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Icon</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="disabledLabel">
      <property name="text">
       <string>Disabled</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="disabledCheckBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
